src/components/TransactionsList.js

```diff
@@ -27,8 +27,14 @@ const TransactionsList = () => {
             </h3>
           </th>
         </tr>
-
-        {"... your code here..."}
+           {props.transactions.map((transaction) =>
+          <tr>
+           <td> {transaction.posted_at} </td>
```

Comment:
Here, I would render out the `Transaction` component from the other file. In general, think about how you might split out your components to keep them small and readable. If you're iterating over a list, it's probably a good idea to create an inner Component to create on the loops